compact/ecal/barrel_interlayers.xml

```diff
@@ -143,13 +143,16 @@
       <dimensions numsides="EcalBarrel_ModRepeat"
         rmin="EcalBarrel_rmin"
         z="EcalBarrel_Calorimeter_length"/>
+      <support thickness="EcalBarrel_Support_thickness" material="Aluminum" vis="EcalBarrelSupportVis"/>
```

Comment:
Since the support is on the outside, and since you're listing the layers from inside to outside here, it may make sense to put the support at line 182. Geometry plugin wouldn't care.

Reply:
Thanks, that makes sense.